src/OctoshiftCLI.IntegrationTests/GhesToGithub.cs

```diff
@@ -83,8 +86,14 @@ await retryPolicy.Retry(async () =>
             await _sourceHelper.CreateGithubRepo(githubSourceOrg, repo2);
         });
 
-        await _targetHelper.RunGeiCliMigration(
-            $"generate-script --github-source-org {githubSourceOrg} --github-target-org {githubTargetOrg} --ghes-api-url {GHES_API_URL} --download-migration-logs", _tokens);
+        // Build the command with conditional option
+        var command = $"generate-script --github-source-org {githubSourceOrg} --github-target-org {githubTargetOrg} --ghes-api-url {GHES_API_URL} --download-migration-logs";
+        if (useGithubStorage)
+        {
+            command += " --use-github-storage true";
+        }
```

Comment:
simply adding the `--use-github-storage` option won't make this work as it's going to be a bit more involved. The problem is this test was designed to work only with an Azure storage so in the constructor it creates a `_blobStroageClient` and on line 77 it resets the blob containres `_targetHelper.ResetBlobContainers()`. Given that this test will be run so many times, it would also be nice to do the same if possible,. If not, we can simply ignore the `ResetBlobContainers` line when using the GitHub storage. 

Additionally we set the `AZURE_STORAGE_CONNECTION_STRING` on line 46 but now that we also want to support gh storage, that dictionary entry should be added only if the `useGithStorage` is `false` and it should be done inside the test method itself.